### ISSUE TITLE
Room booking: impossible to readd dates skipped due to conflicts

### ISSUE THREAD

**Issue by jouvin** [indico/indico#4180]
I had a daily recurring reservation with 5 days. When I did the booking, 3 days were in conflict with another reservation but I choose to get the reservation created anyway. Later, I fixed the conflict (the other reservation was canceled) and I wanted to edit the reservation and readd the missing days (mentioned as "skipped" in the reservation timeline). I didn't manage to do it. It was not possible to extend the reservation as it was already spanning the whole period. And I could not reduce the booking length before the first missing reservation because the corresponding date was in the past (which is fine in itself).

Michel

**Comment by ThiefMaster** [comment]
You are right, this is not possible. Easiest option would be to create a second reservation for those days.

I think adding support to "re-book" skipped days would add A LOT of complexity in case prebookings are required for a room...
